### PR TITLE
first homework done

### PR DESCRIPTION
**Pull request by mrsOwl**
Обладательница самого легкого задания справилась =^_^=

### CODE REVIEW

File: homework.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
-VARIANT = 0
+VARIANT = 1
 
-def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+def generate 
+  i, j, k = rand(1000...10000), rand(1000...10000), rand(1000...10000)
```
Comment (sveredyuk) [inline homework.rb]: this is not ruby :(

Comment (sveredyuk) [reply homework.rb]: think about "don't repeat yourself" and try use less but more readable code

File: homework.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+  
+   "#{i}" + "-" + "#{j}" + "-" + "#{k}"
```
Comment (sergiy-koshoviy) [inline homework.rb]: try to rewrite without '+' operator

File: homework.rb (hunk 1 above)
Comment (sergiy-koshoviy) [inline homework.rb]: 'i', 'j', 'k' is incorrect names. Name of variables, methods and classes should explain context, type, sense and other

File: homework.rb [hunk 3]
```diff
@@ -1,6 +1,15 @@
-VARIANT = 0
+VARIANT = 1
 
-def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+def generate 
+
+  def random_12_numbers
```
Comment (ruslan-369) [inline homework.rb]: don't define method in scope of other method!!!

File: homework.rb [hunk 4]
```diff
@@ -1,6 +1,15 @@
...
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+def generate 
+
+  def random_12_numbers 
+      return rand(1000..10000)
+  end
+
+
+  first_random_four_numbers = random_12_numbers()
```
Comment (ruslan-369) [inline homework.rb]: it's a bad practice in ruby to use brackets in method call if u aren't passing any args

File: homework.rb [hunk 5]
```diff
@@ -1,6 +1,15 @@
...
+  second_random_four_numbers = random_12_numbers()
+  third_random_four_numbers = random_12_numbers()
+
+  return "#{first_random_four_numbers}-#{second_random_four_numbers}-#{third_random_four_numbers}"
```
Comment (ruslan-369) [inline homework.rb]: don't use `return` statement at the end of method body

File: homework.rb [hunk 6]
```diff
@@ -1,6 +1,14 @@
-VARIANT = 0
+VARIANT = 1
 
-def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+
+def random_4_numbers 
+      rand(1000..10000)
```
Comment (sergiy-koshoviy) [inline homework.rb]: 2 space

File: homework.rb [hunk 7]
```diff
@@ -1,6 +1,14 @@
...
+end
+
+def generate 
+  first_random_four_numbers = random_4_numbers
+  second_random_four_numbers = random_4_numbers
+  third_random_four_numbers = random_4_numbers
```
Comment (sergiy-koshoviy) [inline homework.rb]: prev 3 lines are not necessary
you can just call random_4_numbers method in string with interpolation

**Pull request closed without merging**